app/Http/Transformers/Three/SignupTransformer.php

```diff
@@ -25,18 +25,23 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
-        return [
+        $response = [
             'id' => $signup->id,
             'northstar_id' => $signup->northstar_id,
             'campaign_id' => $signup->campaign_id,
             'campaign_run_id' => $signup->campaign_run_id,
             'quantity' => $signup->getQuantity(),
-            'why_participated' => $signup->why_participated,
             'source' => $signup->source,
```

Comment:
Why do we gate showing post `source` and not signup `source`?

Reply:
Mmm yeah, let's do the same here. 👍

Reply:
oh great call @sbsmith86 !